Getting started: package.json `name` must begin with a capital letter

Running through the README instructions, step 1:

https://github.com/jordwalke/CommonML#1-make-a-package-from-scratch
```
$ node -v
v5.1.0

$ node node_modules/CommonML/build.js

Building Root Package /Users/s/tmp/ocaml/MyProject [byte]

Scanning files from /Users/s/tmp/ocaml/MyProject

[ERROR] /Users/s/tmp/ocaml/MyProject/package.json:1 characters 0-0
Invalid package.json for undefined at /Users/s/tmp/ocaml/MyProject/package.json.
package.json `name` must begin with a capital letter.


Writing Package Errors: /Users/s/tmp/ocaml/MyProject/_build_byte/__packageDiagnostics.json
Dependencies scanned and verified, but failed to build

Stack Trace:
------------
 Error: ENOENT: no such file or directory, open '/Users/s/tmp/ocaml/MyProject/_build_byte/__packageDiagnostics.json'
    at Error (native)
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.writeFileSync (fs.js:1224:33)
    at buildTree (/Users/s/tmp/ocaml/MyProject/node_modules/CommonML/build.js:2245:8)
    at whenVerifiedPath (/Users/s/tmp/ocaml/MyProject/node_modules/CommonML/build.js:2450:9)
    at Object.<anonymous> (/Users/s/tmp/ocaml/MyProject/node_modules/CommonML/build.js:2478:3)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
```

--- a/README.md
+++ b/README.md
@@ -150,7 +150,7 @@ Resolving Duplicate Packages:
 Duplicate modules are namespaced automatically, but duplicate packages can only be resolved
 by the package manager - `npm`.
 If two packages depend on two different versions of the same module - there is no solution yet.
-If two packages depend on *compatible* versions of the same module, then `npm dedupde` should take care of it.
+If two packages depend on *compatible* versions of the same module, then `npm dedupe` should take care of it.
 
 Build Parameters:
 -----------------

--- a/README.md
+++ b/README.md
@@ -84,6 +84,7 @@ To make your own package just like the example:
 
     mkdir MyProject && cd MyProject
     echo '{
+      "name": "MyProject",
       "dependencies": {"CommonML": "git://github.com/jordwalke/CommonML.git"},
       "CommonML": {
         "exports": ["MyPublicModule"],

